group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -1446,6 +1459,22 @@ private void throwIfNull(
         }
     }
 
+    /**
+     * Throws an InvalidRequestException if the value is null or empty.
+     *
+     * @param value The value.
+     * @param error The error message.
+     * @throws InvalidRequestException
+     */
+    private void throwIfNullOrEmpty(
+            String value,
+            String error
+    ) throws InvalidRequestException {
+        if (value == null || value.trim().isEmpty()) {
+            throw new InvalidRequestException(error);
+        }
+    }
```

Comment:
This method shows up in the [kip1070](https://github.com/apache/kafka/blob/kip1071/group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java#L1449-L1463) branch from https://github.com/apache/kafka/pull/18069 - I guess with some of the changes it's not showing up as a conflict, just need to be aware for after the merge.